src/styles/card.css

```
@@ -0,0 +1,96 @@
+.card {
+  width: 200px;
+  height: 406px;
+  padding: 32px 16px 16px;
+
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+}
+
+.card__logo {
+  display: block;
+  width: 160px;
+  height: 134px;
+  margin: 0 auto 40px;
+}
+
+.card__title {
+  display: flex;
+  margin-bottom: 4px;
+}
+
+.card__link {
+  font-size: 12px;
+  line-height: 18px;
+  font-weight: 500;
+  text-decoration: none;
+  color: #060b35;
+}
+
+.card__code {
+  margin-bottom: 16px;
+
+  font-size: 10px;
+  line-height: 14px;
+  font-weight: 400;
+  color: #616070;
+}
+
+.card__rating {
+  display: flex;
+  justify-content: space-between;
+  margin-bottom: 24px;
+}
+
+.card__value {
+  display: flex;
+  justify-content: space-between;
+  margin-bottom: 16px;
+}
+
+.card__review {
+  align-self: center;
+
+  font-size: 10px;
+  line-height: 14px;
+  font-weight: 400;
+}
+
+.card__price {
+  font-size: 12px;
+  line-height: 18px;
+  font-weight: 400;
+  color: #616070;
+}
+
+.card__cost {
+  font-size: 16px;
+  line-height: 18px;
+  font-weight: 700;
+}
+
+.card__button {
+  display: block;
+  width: 166px;
+  height: 40px;
+
+  font-size: 14px;
+  line-height: 40px;
+  font-weight: 700;
+  color: #fff;
+  text-decoration: none;
+  text-transform: uppercase;
+  text-align: center;
+
+  background-color: #00acdc;
+  border-radius: 5px;
+  border: 1px solid transparent;
+
+  transition: all 0.3s;
```

It would be best if you explicitly named all properties that will change on transition
```suggestion
  transition: 0.3s property1 ease, 0.3s property2 ease;
```